db_stress_tool/db_stress_shared_state.h

```diff
@@ -43,11 +43,8 @@ class StressTest;
 // State shared by all concurrent executions of the same benchmark.
 class SharedState {
  public:
-  // indicates a key may have any value (or not be present) as an operation on
-  // it is incomplete.
-  static constexpr uint32_t UNKNOWN_SENTINEL = 0xfffffffe;
-  // indicates a key should definitely be deleted
-  static constexpr uint32_t DELETION_SENTINEL = 0xffffffff;
+  // Used to generate random value base for non-NonBatchedOpsStressTest
+  static constexpr uint32_t RANDOM_VALUE_BASE_GENERATOR = 0xfffffffe;
```

Comment:
I think they are used in batched_op_stress too https://github.com/facebook/rocksdb/blob/main/db_stress_tool/batched_ops_stress.cc#L35. Also it was suggested to keep it in https://github.com/facebook/rocksdb/pull/11058#discussion_r1123872513 - let me know though if any change in your opinion/thoughts since then :)

Reply:
I think you can evaluate whether it's useful or not. The purpose of the linked comment is not about keeping RANDOM_VALUE_BASE_GENERATOR, btw. The purpose is to say we can generate any valid value since it'll be rolled back anyways.

Reply:
oh okay - I will see whether we can remove or localize it to batched_op_stress

Reply:
Fixed by localizing it to batched_op_stress

Reply:
Does this mean we can't remove it? It looks arbitrary since 0xfffffffe and 0xffffffff are no longer sentinel values

Reply:
will fix by removing it - I didn't realized batched_op_stress does not need RANDOM_VALUE_BASE_GENERATOR at all (even before my PR)